test/integration/controllermanager/seed/backupbucketscheck/backupbucketscheck_suite_test.go

```diff
@@ -110,14 +107,28 @@ var _ = BeforeSuite(func() {
 	})
 	Expect(err).NotTo(HaveOccurred())
 
+	By("setting up field indexes")
+	Expect(indexer.AddBackupBucketSeedName(ctx, mgr.GetFieldIndexer())).To(Succeed())
+
 	fakeClock = &testclock.FakeClock{}
```

Comment:
This test is probably going to be flaky due to the same reasons as https://github.com/gardener/gardener/pull/6619#issuecomment-1237233917. Perhaps it makes sense to change this to  `fakeClock = testclock.NewFakeClock(time.Now())` directly in this PR instead of opening a new one. WDYT?